docs/guides/building-with-components.md

```diff
@@ -42,7 +42,7 @@ This is the starting point for our scene:
     <img id="sechelt-thumb" src="img/thumb-sechelt.png">
   </a-assets>
 
-  <!-- 360-degree image. -->
+  <!-- 360° image. -->
```

Comment:
i usually don't have unicode characters in code. might flub up some editors if they're not configured